templates/layout/default.php

```diff
@@ -42,7 +42,7 @@
         </div>
         <div class="top-nav-links">
             <a target="_blank" rel="noopener" href="https://book.cakephp.org/4/">Documentation</a>
-            <a target="_blank" rel="noopener" href="https://api.cakephp.org/4/">API</a>
+            <a target="_blank" rel="noopener" href="https://api.cakephp.org/4.1/">API</a>
```

Comment:
Additionally, the existing URL should probably 301 to the latest.

Reply:
Hmm yes. that is maybe also what I expect. I see this also don't work for /3/